src/components/OrderBookWidget.tsx

```diff
@@ -1,12 +1,40 @@
 import React, { useEffect, useRef } from 'react'
+import BigNumber from 'bignumber.js'
 import styled from 'styled-components'
-import { TokenDetails, Network } from 'types'
+
 import * as am4core from '@amcharts/amcharts4/core'
 import * as am4charts from '@amcharts/amcharts4/charts'
 import am4themesSpiritedaway from '@amcharts/amcharts4/themes/spiritedaway'
-import { getNetworkFromId, safeTokenName } from '@gnosis.pm/dex-js'
+
 import { dexPriceEstimatorApi } from 'api'
 
+import { getNetworkFromId, safeTokenName, formatSmart, toBN } from 'utils'
+
+import { TEN_BIG_NUMBER } from 'const'
+
+import { TokenDetails, Network } from 'types'
+
+function checkPriceAndConvert(price: number): string {
+  // prices come in as type number so need to be converted
+  // some of the WEI values are so small we need to cast to Wei twice
+  const priceAsBigNumber = new BigNumber(price)
+  const calculatedPriceAsBnWei = toBN(
+    priceAsBigNumber
+      .times(TEN_BIG_NUMBER.pow(new BigNumber('18')))
```

Comment:
> nice.
> 
> Also confused by the comment Leandro pointed out.

mm because some values are so small as Wei: `0.003123897234` for example that can't be converted to `BN` and `formatSmart` requires a `BN` amount, not `BIgNumber` open to better options